articles/contributing/docs.md

```diff
@@ -46,42 +46,46 @@ When we compile each release of the Quantum Development Kit, these comments are
 > Please make sure to not manually edit the generated API documentation, as these files are overwritten with each new release.
 > We value your contribution to the community, and want to make sure that your changes continue to help users release after release.
 
-For example, consider an operation `PrepareTrialState(angles : Double[], register : Qubit[]) : Unit`.
-A documentation comment should help a user learn how to interpret `angles`, what the operation assumes about the initial state of `register`, what the effect on `register` is, and so forth.
+For example, consider the operation `GreaterThan(xs : LittleEndian, ys : LittleEndian, result : Qubit) : Unit`.
+A documentation comment should help a user learn how to interpret `xs` and `ys`, what the operation assumes about the initial state of `result`, what the effect on `result` is, and so forth.
 Each of these different pieces of information can be provided to the Q# compiler by a specially named Markdown section in the documentation comment.
-For the example of `PrepareTrialState`, we might write something like the following:
+For the example of `GreaterThan`, we might write something like the following:
 
 ```qsharp
 /// # Summary
-/// Given a register of qubits, prepares them in a trial state by rotating each
-/// independently.
+/// Applies a greater-than comparison between two integers encoded into
+/// qubit registers, flipping a target qubit based on the result of the
+/// comparison.
 ///
 /// # Description
-/// This operation prepares the input register by performing a
-/// $Y$ rotation on each qubit by an angle given in `angles`.
+/// Carries out a strictly greater than comparison of two integers $x$ and $y$, encoded
+/// in qubit registers xs and ys. If $x > y$, then the result qubit will be flipped,
+/// otherwise the result qubit will retain its state.
 ///
 /// # Input
-/// ## angles
-/// An array of parameters
-/// ## register
-/// A register of qubits initially in the $\ket{00\cdots0}$ state.
+/// ## xs
+/// LittleEndian qubit register encoding the first integer $x$.
```

Comment:
Here, the type information (namely, `LittleEndian`) is provided by the documentation generation tool and is thus redundant. Thus, we can instead focus on what that type means to the user: that it specifies the representation used for encoding the integer into qubits.
```suggestion
/// A register encoding the first integer $x$ in a little-endian representation.
```